lib/functions/image/rootfs-to-image.sh

```diff
@@ -58,8 +58,8 @@ PRE_UPDATE_INITRAMFS
 	# DEBUG: print free space
 	local freespace=$(LC_ALL=C df -h)
 	echo -e "$freespace" >> $DEST/${LOG_SUBPATH}/debootstrap.log
-	display_alert "Free SD cache" "$(echo -e "$freespace" | grep $SDCARD | awk '{print $5}')" "info"
-	display_alert "Mount point" "$(echo -e "$freespace" | grep $MOUNT | head -1 | awk '{print $5}')" "info"
+	display_alert "Free SD cache" "$(echo -e "$freespace" | awk "\$6==\"${SDCARD}\" {print \$5}")" "info"
+	display_alert "Mount point" "$(echo -e "$freespace" | awk "\$6==\"${MOUNT}\" {print \$5}")" "info"
```

Comment:
An awk script surrounded by double quotes is potentially dangerous. I can't approve.
Instead, pass the variable to the script using the `-v` parameter:
```bash
display_alert "Mount point" "$(echo -e "$freespace" | awk -v m="${MOUNT}" '{if($6 == m) print $5}')" "info"
```